test/cli/test_all.rb

```diff
@@ -0,0 +1,2 @@
+# NOTE: run the test suite with: $ bundle exec ruby -I lib:test test/cli/test_all.rb -v
```

Comment:
It'd be good to make this a target in the Makefile:

```Make
.PHONY: test-cli
test-cli:
    ruby -I lib:test test/cli/test_all.rb -v
```

(That last line should be tab indented, but GitHub doesn't let you input tabs on their website).